src/createTRPCJotai.ts

```diff
@@ -135,7 +135,7 @@ type QueryResolver<TProcedure extends AnyProcedure, TClient> = {
       ProcedureArgs<TProcedure['_def']>[1] & { disabledOutput?: undefined }
     >,
     getClient?: (get: Getter) => TClient,
-  ): Atom<Promise<inferProcedureOutput<TProcedure> | undefined>>;
+  ): WritableAtom<Promise<inferProcedureOutput<TProcedure>>, [], void>;
```

Comment:
Should we keep `undefined`, no?
```suggestion
  ): WritableAtom<Promise<inferProcedureOutput<TProcedure> | undefined>>, [], void>;
```